### PR TITLE
feat: add CLIC theme

### PR DESCRIPTION
**Pull request by NoeTerrier**
Add CLIC theme and remove about content.

### CODE REVIEW

**Comment by NoeTerrier** [comment]
Maybe check if the favicon works well :)

File: docker-compose.yaml [hunk 1]
```diff
@@ -22,5 +22,5 @@ services:
       - 1234:1234
     environment:
       API_URL: http://api:3000
-      NEXT_PUBLIC_API_URL: http://localhost:3000
+      NEXT_PUBLIC_API_URL: http://localhost:1234/crabfit
```
Comment (Lumycea) [inline docker-compose.yaml]: What does this mean ? The API is located a `http://localhost:3000` in the dev environment, `http://localhost:1234/crabfit` is the url of the frontend.

File: frontend/src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -11,25 +7,12 @@ interface FooterProps {
...
-        href="https://ko-fi.com/A06841WZ"
-        target="_blank"
-        rel="noreferrer noopener payment"
-        style={{ whiteSpace: 'nowrap' }}
-      >{t('donate.button')}</Button>
-    </footer>
-}
+  return <footer className={makeClass(styles.footer, isSmall && styles.small)}>
+    <p>
+      Based on <a href="https://github.com/GRA0007/crab.fit">Crab Fit</a> from <a href="https://bengrant.dev/"> Ben Grant</a> for custom style and content of pages.
```
Comment (Lumycea) [inline frontend/src/components/Footer/Footer.tsx]:
```suggestion
      Based on <a href="https://github.com/GRA0007/crab.fit">Crab Fit</a> from <a href="https://bengrant.dev/"> Ben Grant</a>. 
```

I find the formulation unclear. The original repo contains the logic, and the fork was created to change the UI.

**Review by Lumycea — CHANGES_REQUESTED** [review]
Nice UI :)

There is a `NEXT_PUBLIC_FRONTEND_URL` to refer to the frontend base url in the javascript, but this doesn't work from the non js files, we may want to investigate how to improve that.

File: docker-compose.yaml (hunk 1 above)
Comment (NoeTerrier) [reply docker-compose.yaml]: yep my bad I revert this.

**Review by Lumycea — APPROVED** [review]

**Pull request merged**